frontend/src/components/account/EditAccountPage.js

```diff
@@ -594,7 +594,9 @@ export default function EditAccountPage({
                 </Tooltip>
               )}
             </Typography>
-            <TextField fullWidth value={i.value} multiline onChange={handleChange} />
+            {i.required? 
+              <TextField required fullWidth value={i.value} multiline onChange={handleChange} />
+              : <TextField fullWidth value={i.value} multiline onChange={handleChange} />}
```

Comment:
```suggestion
              <TextField required={i.required} fullWidth value={i.value} multiline onChange={handleChange} />
```
I'd use this annotation as it saves 2 lines in this context.